src/components/Contact/Follow.js

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from "react";
```

Comment:
Follow.js: Code is clean and clearly presented. Any particular reason for the change from `import React, { Component } from 'react';` to `import React, { Component } from "react";`?